bin/ghsec-jira

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+// The following way to require composers autoload.php is borrowed
+// from https://github.com/sebastianbergmann/phpunit
+foreach ([__DIR__ . '/../../autoload.php', __DIR__ . '/../vendor/autoload.php', __DIR__ . '/vendor/autoload.php'] as $file) {
```

Comment:
The third option is not really needed. First covers installing this project in another project (which I guess covers global installs too), but is that really needed? Second case covers cloning the project and doing a `composer install`. And the last? I really don't know when you'd have that case.

Reply:
I'm pretty sure @achton has it from the code I wrote for https://github.com/reload/lumturio-jira. And I stole it from phpunit (as the comment says). I'm not sure / don't remember what use cases it's supposed to cover. I think one of them also covers putting all the stuff into a PHAR file.

Reply:
AFAIK, a phar is just an archive, so relative paths work the same.

Reply:
Ok, dunno then. Except phpunit does it :)

Reply:
PHPUnit uses an XML config file. I rest my case.